model/funque_float.json

```diff
@@ -13,20 +13,39 @@
         "model_type": "LIBSVMNUSVR",
         "feature_opts_dicts": [
             {
+                "enable_spatial_csf": false,
                 "enable_resize": true,
-                "vif_levels": 2
-            },
-            {},
-            {},
-            {},
-            {}
+
+                "spatial_csf_filter": 5,
+                "wavelet_csf_filter": 1,
+
+                "vif_levels": 4,
+                "adm_levels": 4,
+                "ssim_levels": 4,
+                "ms_ssim_levels": 4,
+                "strred_levels": 4
+            }
         ],
         "feature_names": [
-            "FUNQUE_feature_vif_scale0_score",
-            "FUNQUE_feature_vif_scale1_score",
-            "FUNQUE_feature_motion_score",
-            "FUNQUE_feature_adm2_score",
-            "FUNQUE_float_ssim"
+            "FUNQUE_feature_vif_score",
+            "FUNQUE_feature_vif_scale0_score", "FUNQUE_feature_vif_scale1_score",
+            "FUNQUE_feature_vif_scale2_score", "FUNQUE_feature_vif_scale3_score",
+
+            "FUNQUE_feature_adm_score",
+            "FUNQUE_feature_adm_scale0_score", "FUNQUE_feature_adm_scale1_score",
+            "FUNQUE_feature_adm_scale2_score", "FUNQUE_feature_adm_scale3_score",
+
+            "FUNQUE_feature_ssim_scale0_score", "FUNQUE_feature_ssim_scale1_score",
+            "FUNQUE_feature_ssim_scale2_score", "FUNQUE_feature_ssim_scale3_score",
+
+            "FUNQUE_feature_strred_scale0_score", "FUNQUE_feature_strred_scale1_score",
+            "FUNQUE_feature_strred_scale2_score", "FUNQUE_feature_strred_scale3_score",
+            "FUNQUE_feature_ms_ssim_mean_scale0_score", "FUNQUE_feature_ms_ssim_mean_scale1_score",
+            "FUNQUE_feature_ms_ssim_mean_scale2_score", "FUNQUE_feature_ms_ssim_mean_scale3_score",
+            "FUNQUE_feature_ms_ssim_cov_scale0_score", "FUNQUE_feature_ms_ssim_cov_scale1_score",
+            "FUNQUE_feature_ms_ssim_cov_scale2_score", "FUNQUE_feature_ms_ssim_cov_scale3_score",
+            "FUNQUE_feature_ms_ssim_mink3_scale0_score", "FUNQUE_feature_ms_ssim_mink3_scale1_score",
```

Comment:
the `_cov_` variants seem to return odd results (looks like a `1 -` might be missing). However I intend to delete those in any case so it doesn't really matter.